utils/gen_onnx_mlir.py

```diff
@@ -295,6 +295,7 @@
     'ReverseSequence',
     'ScatterND',
     'SpaceToDepth',
+    'SequenceInsert',
```

Comment:
Add in alphabetical order.

Reply:
VSCode let you highlight code segment and then right click to order.. useful to me :-)

Reply:
Fixed